lambda_src/drivers/process_https.py

```diff
@@ -118,19 +135,33 @@ def process_row(
 
     LOG.debug('Starting pagination.')
     while next_url:
-        LOG.debug(f'next_url is {next_url}.')
+        LOG.info('next_url is %s.', next_url)
         req = Request(next_url, method=req_method, headers=req_headers, data=req_data)
         links_headers = None
 
         try:
             LOG.debug(f'Making request with {req}')
```

Comment:
This doesn't seem to print anything important it seems -- the req object has no special `__repr__` or `__str__`, and it only ends up printing the class and the memory address of the object.

Something like `<urllib.request.Request object at 0x7fe94f185310>`. This could be removed maybe?

Reply:
delete